### PR TITLE
Support ContinueInitializationOnError for Acceptors

### LINKED ISSUE #293: ContinueInitializationOnError only works for Initiators
**Describe the bug**
We have a setting `ContinueInitializationOnError` which could be used when a single session should not prevent the whole connector (with multiple sessions) from starting. However, it is only implemented for Initiators.

https://github.com/quickfix-j/quickfixj/blob/a8725ad9a849dbfcb4c8f220538daa721029c101/quickfixj-core/src/main/java/quickfix/mina/initiator/AbstractSocketInitiator.java#L178-L183

**Expected behavior**
This should also work for Acceptors.

### PR DESCRIPTION
**Pull request by chrjohn**
Fixes #293

### CODE REVIEW

File: quickfixj-core/src/main/java/quickfix/mina/acceptor/AbstractSocketAcceptor.java [hunk 1]
```diff
@@ -227,11 +229,20 @@ private void createSessions(SessionSettings settings) throws ConfigError, FieldC
...
-                    descriptor.acceptSession(session);
-                    allSessions.put(sessionID, session);
+                try {
+                    AcceptorSocketDescriptor descriptor = getAcceptorSocketDescriptor(settings, sessionID);
+                    if (!isTemplate) {
+                        Session session = sessionFactory.create(sessionID, settings);
+                        descriptor.acceptSession(session);
+                        allSessions.put(sessionID, session);
+                    }
+                } catch (Throwable t) {
```
Comment (chrjohn) [inline quickfixj-core/src/main/java/quickfix/mina/acceptor/AbstractSocketAcceptor.java:239]: Not a big fan of catching general `Throwable` but made code as similar as possible to AbstractSocketInitiator. Could be changed later.

**Comment by chrjohn** [comment]
NB: LGTM seem to finally have disabled JavaScript analysis. :tada:

**Comment by chrjohn** [comment]
@the-thing do you have a minute for a review? No big change but since I want to merge this in before the release I'd rather have someone other take a look.
Thanks :)

File: quickfixj-core/src/test/java/quickfix/MultiAcceptorTest.java [hunk 2]
```diff
@@ -111,6 +129,26 @@ public void testMessageSentOnWrongAcceptor() throws Exception {
...
+        int freePort3 = AvailablePortFinder.getNextAvailable();
+
+        Acceptor acceptor = createAcceptor(freePort1, freePort2, freePort3, true);
+        acceptor.start();
+
+        List<SessionID> acceptorSessions = acceptor.getSessions();
+        assertEquals(2, acceptorSessions.size());
+        SessionID session1 = new SessionID(FixVersions.BEGINSTRING_FIX42, "ACCEPTOR-" + 1, "INITIATOR");
+        SessionID session3 = new SessionID(FixVersions.BEGINSTRING_FIX42, "ACCEPTOR-" + 3, "INITIATOR");
+        assertTrue(acceptorSessions.contains(session1));
```
Comment (the-thing) [inline quickfixj-core/src/test/java/quickfix/MultiAcceptorTest.java:146]: `assertFalse(acceptorSessions.contains(session2));` would be good.

Comment (chrjohn) [reply quickfixj-core/src/test/java/quickfix/MultiAcceptorTest.java:148]: Thanks, done.

**Pull request merged**